shell/platform/linux/fl_view.cc

```diff
@@ -239,6 +243,8 @@ static void fl_view_realize(GtkWidget* widget) {
     g_warning("Failed to start Flutter engine: %s", error->message);
     return;
   }
+
+  fl_settings_plugin_start(self->settings_plugin);
```

Comment:
Is there a reason to put this on the view rather than the engine?

Reply:
I couldn't decide where to put it, so I put it with the other plugins for consistency. It doesn't requite anything specific from the view, but then neither does the platform plugin.

I think if we have some of them attached to the engine and some attached to the view then we need to do some renaming to make it clearer what is different about them. The name plugin is already confusing as they really aren't plugins at all.

Reply:
Yes, that's something that propagated from mobile, but in retrospect I regret following that naming. We could consider a mass change at some point.

macOS has all its "plugins" on the view class for legacy reasons; in practice only the ones related to the view (e.g., text input) should live there. A headless engine should still have the right time format, for instance.

Reply:
I've moved this plugin to the engine.